test/telemetry_helpers.erl

```diff
@@ -0,0 +1,27 @@
+-module(telemetry_helpers).
+
+-define(HANDLER, telemetry_handler).
+-define(CONFIG, #{dummy_config => true}).
+-define(CONFIG_MATCH, #{dummy_config := true}).
+
+-export([start/1, stop/0, get_calls/1]).
+
+start(TelemetryEvents) ->
+    meck:new(?HANDLER, [non_strict, no_link]),
+    meck:expect(?HANDLER, handler, ['_', '_', '_', '_'], ok),
+    application:start(telemetry),
+    TelemetryHandler = fun ?HANDLER:handler/4,
+    telemetry:attach_many(?HANDLER, TelemetryEvents, TelemetryHandler, ?CONFIG).
+
+stop() ->
+    meck:reset(?HANDLER).
```

Comment:
this is quite a confusing name for the function. there should be a separate stop function and a separate reset function.